lib/matplotlib/axes/_axes.py

```diff
@@ -4655,6 +4656,8 @@ def reduce_C_function(C: array) -> float
             DATA_PARAMETER_PLACEHOLDER
 
         **kwargs : `~matplotlib.collections.PolyCollection` properties
+            If *xscale* or *yscale* is set to 'symlog',
+            additional `~matplotlib.scale.SymmetricalLogTransform` parameters.
```

Comment:
Hmm, not sure if this is the way we want to expose things @timhoffm?

Reply:
Mix those parameters into the generic kwargs gets too messy. Instead we should do one of the following:

- Add a new parameters `xscale_kwargs`, `yscale_kwargs`. Note that this is named scale-agnostic, so that we can use it for parameters of other scales in the future as well. Also we should directly include the possibility to have separate parameters for x and y.
  i.e. `hexbin(..., xscale='symlog', xscale_kwargs={'base': 10, 'linthresh': 2, 'linscale': 1}), ...)`
- Include the values into the `xscale` / `yscale` parameters.
  i.e. `hexbin(..., xscale=('symlog', {'base': 10, 'linthresh': 2, 'linscale': 1}), ...)`


I'm inclined towards the latter because it keeps the related information bound together even more closely. I'll propose a suggestion for the docstring.